launch/Jenkinsfile

```diff
@@ -26,27 +25,21 @@ try {
     parallel "openhab2-addons": {
         node {
             deleteDir()
-            installTychoVersionsHotfix()
             releaseOpenHab2Component('openhab2-addons', true)
+        }    
+    }, "openhab2-extra-addons": {
```

Comment:
Are you sure this works? These projects depend on a parent that is build in openhab-addons which is released in parallel by the other node:

https://github.com/openhab/org.openhab.binding.zwave/blob/c6ffb28e6c7e4bf37779bc4b740778fec9eff346/pom.xml#L6

Reply:
Good point! I thought it can be fetched from nexus, but probably it won't work for the first "next version snapshotbuild" test.

Reply:
I once had a look if it were possible to release them all in parallel and stumbled upon this. ;-)

Do you know if all Maven plugins used during the release are thread safe? It might speed up releases considerably if you can use a parallel release build (e.g. adding `-T 1C` to mvn commands). 

I do everything with these parallel builds including releases on another project and most of the parallelization issues that I know of for openHAB have been solved except for SAT (https://github.com/openhab/static-code-analysis/pull/365).

Reply:
No, I don't know. But I'd vote against a parallel release build. Given the assumption that it doesn't cause technical issues (which is already brave), it's not a good thing for releasebuild. With parallel builds, you gain speed, but you loose traceability and sometimes also reproducibility. For CI snapshotbuilds, parallel builds can be great to reduce the time developers get feedback about their changes. But for releasebuilds, traceability is more important than speed.

Reply:
Initially you might run in to some issues with these parallel builds of course. But like everything, the more you use them the better you get with them. Then all issues are fixed and they gain full confidence. After that you can no longer imagine living in a world without them. ;-)